pkg/syncer/metadatasyncer.go

```diff
@@ -518,6 +523,12 @@ func pvcDeleted(obj interface{}, metadataSyncer *metadataSyncInformer) {
 		return
 	}
 
+	// Verify if pv is a vsphere volume and migration flag is set to false
+	if pv.Spec.VsphereVolume != nil && !metadataSyncer.configInfo.Cfg.FeatureStates.CSIMigration {
+		log.Warnf("PVCDeleted: Unable to delete vSphere volume claim %s when migration feature flag is false", pvc.Name)
```

Comment:
Can we change this to
log.Warnf("PVCDeleted: Can not delete metadata for migrated vSphere volume claim %s when migration feature flag is false", pvc.Name) 

add `metadata` in the message.

Reply:
I have updated the logs message to be consistent with controller. Let me know if that is fine. If not, I can change as per your suggestion